Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -1985,11 +1997,33 @@ bool AccessibilityNodeObject::isDataTable() const
 AXCoreObject::AccessibilityChildrenVector AccessibilityNodeObject::rowHeaders()
 {
     AccessibilityChildrenVector headers;
-    auto rowsCopy = rows();
-    for (const auto& row : rowsCopy) {
-        if (RefPtr header = row->rowHeader())
-            headers.append(header.releaseNonNull());
+
+    if (isTableRow() || isTable()) {
+        auto rowsCopy = rows();
+        for (const auto& row : rowsCopy) {
+            if (RefPtr header = row->rowHeader())
+                headers.append(header.releaseNonNull());
+        }
+    } else if (isTableCell()) {
+        RefPtr parent = parentTable();
+        if (!parent)
+            return headers;
+
+        auto rowRange = rowIndexRange();
+        auto colRange = columnIndexRange();
```

Comment:
I think WebKit prefers to explicitly spell out most names, e.g. `columnRange` rather than `colRange`